app/main.py

```diff
@@ -1 +1,53 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    # Exclude input without the command "mv"
+    if command.split()[0] != "mv":
```

Comment:
Don't use `split()` twice. Create a variable that will store the result of `split()`